lib/resources/example_json/r4_examples_communication.rb

```diff
@@ -0,0 +1,98 @@
+module Cerner
+  module Resources
+
+    R4_COMMUNICATION_ENTRY ||= {
+      "resourceType": "Communication",
+      "id": "C-1123_0",
+      "meta": {
+          "versionId": "0",
+          "lastUpdated": "2020-07-16T21:31:35.457Z"
+      },
+      "text": {
+          "status": "generated",
+          "div": "<div xmlns=\"http://www.w3.org/1999/xhtml\"><p><b>Communication</b></p></div>"
```

Comment:
I bet the div isnt being rendered correctly

Reply:
> I bet the div isnt being rendered correctly

I copied the response from Postman. Should I transform the response by something?

Reply:
Check out what the other resources do to their divs :)

Reply:
https://github.com/cerner/fhir.cerner.com/pull/433/commits/467649b127ef86e0c5069186027e3eaf632fb018